algorithms/branching.py

```diff
@@ -90,8 +90,11 @@ def __branching(G: 'Graph', colors: 'Dict[Int, List[Vertex]]', C: 'Int', D: 'Lis
             if y.graph_label == 2 and y.label == y0.label:
                 y.colornum = max_colornum
                 I_copy.append(y)
-                num_isomorphisms += count_isomorphisms(G_copy, D_copy, I_copy, count_flag)
+                num_isomorphisms += count_isomorphisms(G_copy, D_copy, I_copy, count_flag, color_refinement)
                 if not count_flag and num_isomorphisms > 0:
                     return True
                 break
-    return num_isomorphisms
+    if not count_flag:
```

Comment:
Waarvoor is dit nodig? Dat ging toch al goed met dat het True weergeeft voor groter dan nul en False voor nul? Of is dit meer voor het overzicht?

Reply:
Deze if statement staat ook op regel 94.
if not count_flag and num_isomorphisms >0:
     return True

Qua overzicht mag, maar ik weet niet of regel 98 kan worden uitgevoerd. Het lijkt erop dat r.95 altijd eerder is met return True;

Reply:
Het is inderdaad meer voor het overzicht. Want in de integratietest heb ik nu ook standaard zitten dat hij het aantal isomorphisms print. Alleen deze methode gaf altijd een integer terug, omdat True + integer, die integer + 1 is. Dus om het in de csv files mooi te laten lijken heb ik dit toegevoegd.